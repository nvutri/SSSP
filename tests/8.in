c random graph n=32 m=128 lo=0 hi=65536 seed=98272738
p sp 32 128
a 31 32 37527
a 32 18 37786
a 18 5 57362
a 5 14 25168
a 14 9 48832
a 9 13 28494
a 13 16 7599
a 16 23 53475
a 23 12 47009
a 12 26 5574
a 26 22 41627
a 22 7 51711
a 7 10 64116
a 10 2 4401
a 2 4 45979
a 4 15 52408
a 15 8 35782
a 8 25 22499
a 25 19 61133
a 19 28 60855
a 28 20 10014
a 20 30 53971
a 30 11 5390
a 11 27 59690
a 27 17 65362
a 17 29 12486
a 29 1 49104
a 1 21 58342
a 21 24 62481
a 24 6 51280
a 6 3 34502
a 3 31 571
a 4 26 28255
a 28 8 46250
a 2 8 62927
a 23 20 6832
a 23 29 13058
a 11 19 49629
a 8 15 12993
a 30 9 62212
a 4 25 34126
a 5 4 50404
a 19 23 41648
a 7 14 17638
a 26 19 36601
a 11 24 44450
a 16 21 11307
a 31 8 31605
a 27 15 7078
a 23 20 28294
a 26 9 35113
a 24 17 64155
a 17 21 61806
a 19 12 30752
a 15 16 58800
a 4 11 22033
a 17 11 27769
a 26 12 28724
a 5 18 1551
a 16 7 27338
a 20 29 59052
a 29 31 57574
a 27 28 24199
a 1 17 4003
a 26 8 49003
a 26 23 10216
a 29 2 27065
a 25 14 50873
a 6 23 22796
a 23 18 54553
a 23 6 319
a 16 24 10255
a 20 17 20021
a 1 20 46955
a 29 8 32968
a 30 16 42047
a 1 5 337
a 25 22 65220
a 25 28 39026
a 8 28 49864
a 19 7 478
a 6 20 28374
a 14 7 10882
a 18 2 6921
a 31 7 22667
a 27 26 48805
a 31 27 46676
a 17 19 17984
a 25 13 58122
a 24 32 51951
a 13 27 836
a 8 24 14717
a 28 5 13090
a 16 17 38444
a 3 10 40031
a 6 23 24333
a 20 2 3917
a 21 29 21168
a 16 26 15260
a 17 19 12094
a 29 24 31339
a 14 18 61900
a 15 1 50623
a 25 30 42171
a 2 30 15977
a 30 32 64230
a 20 12 34006
a 23 10 54131
a 11 14 54877
a 22 21 40233
a 28 15 53738
a 22 21 32203
a 14 16 55298
a 14 12 10454
a 14 2 47137
a 3 19 28366
a 12 8 40519
a 25 4 15072
a 15 2 11772
a 10 21 14854
a 19 16 44831
a 8 15 30152
a 26 8 55124
a 32 4 22936
a 8 23 51421
a 2 28 5706
a 20 13 50874
a 6 20 33560
