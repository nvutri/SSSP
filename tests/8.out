1 0
2 8809
3 86570
4 50741
5 337
6 52068
7 16575
8 49457
9 74337
10 80691
11 30176
12 35959
13 75403
14 25505
15 68253
16 60928
17 4003
18 1888
19 16097
20 24529
21 58342
22 83160
23 51749
24 47828
25 71956
26 41533
27 76239
28 14515
29 16489
30 24786
31 74063
32 89016
